akka-persistence/src/test/scala/akka/persistence/PersistentActorStashingSpec.scala

```diff
@@ -166,7 +166,7 @@ abstract class PersistentActorStashingSpec(config: Config) extends PersistenceSp
 
       cmds foreach (persistentActor ! _)
       persistentActor ! GetState
-      expectMsg(evts)
+      expectMsg(evts.toList)
```

Comment:
This is a totally weird one! Without the change there is a classcastexception here!